README.md

```diff
@@ -140,26 +140,31 @@ This package contains some launch files, depending on the nodes you want to run.
 - **`robot_controls.launch.py`**: this launch file allows to run the `controller manager` node for ros2_control and the nodes for the single controllers (one for each part).
 - **`circle_demo.launch.py`** and **`grasp_demo.launch.py`**: as the name suggests, they are two examples of commanding the robot in the cartesian space using `torso + right_arm` as planning group.
 
-
 Before running the nodes, make sure that your environment variable `YARP_ROBOT_NAME` is properly set with the name of your robot. If not, set it for each shell according to the chosen model, for example:
 
 ```shell
-export YARP_ROBOT_NAME="icub"
+export YARP_ROBOT_NAME="iCubGenova11"
+```
+
+or, for simulated models:
+
+```shell
+export YARP_ROBOT_NAME="iCubGazeboV2_5"
 ```
 
-### icub_moveit_config
+### xcub_icub_moveit_config
```

Comment:
Isn't better `icub_moveit_config` ? why we need the xcub prefix? maybe I missed something @traversaro @martinaxgloria

Reply:
I kept the `xcub_` prefix also for icub/ergocub config packages to be compliant with the other packages. I know, it is redundant. Since this is robot-specific, we can avoid using the prefix in these cases and call them <robot_name>_moveit_config. @traversaro what do you think about?

Reply:
Both `xcub_icub_moveit_config` and `icub_moveit_config` are fine for me, I do not have any preference.